_posts/2020-12-24-how-to-start-with-foreman

```diff
@@ -0,0 +1,399 @@
+and click Register Content Host. Follow the instructions on this page.
+
+To make this easier, you can also use [bootstrap.py](https://github.com/Katello/katello-client-bootstrap),
+that accepts many options. See it's readme for more details. The linked repository
+also contains ansible playbook, that can be used to automate a subscription of
+multiple hosts. 
+
+### Importing VMs from compute resource
+
+Foreman can scan many virtualization hypervisors and cloud providers. Each hypervisor
+and cloud provider offers different capabilities. Foreman abstracts these differences
+and as much as possible. In order to start using some virtualization backend, a compute
+resources must be defined. Compute resource represents the hypervisor or cloud.
+
+Some providers are shipped with Foreman but more can be added by installing plugins.
+As of today, these are part of Foreman core:
+
+* Amazon EC2
+* Google Compute Engine
+* Libvirt
+* OpenStack
+* oVirt / RHV
+* VMware (with vSphere only)
+
+Some popular compute resource types added by plugins are: Azure, Digital Ocean,
+HyperV, Scaleway, Xen. If your provider is not listed here, you can
+write the new plugin.
+
+Let's say you start using Foreman with some existing VM infrastructure. The easiest
+way to start is connecting Foreman to your virtualization and import all VMs from
+there. First we need to create a new compute resource object.
+
+Navigate to *Infrastructure -> Compute resources* and click *Create Compute Resource*
+button. Give it some name and select a provider type. If you don't see any provider,
+you need to install some. While some providers are part of Foreman core, we
+ship them in separate RPM packages, so users can install only those, that are
+interesting in their environment. In order to see, what providers you have
+installed, navigate to *Administer -> About -> Available Providers*.
+
+<p float='left' align='center'>
+  <img src="/static/images/blog_images/2020-12-24-how-to-start-with-foreman/cr_providers.png" width="1000" />
+</p>
+
+Here's a table with package name for each core provider
+
+<table class="table table-bordered table-condensed">
+  <tr>
+    <th>Virtualization</th>
+    <th>RPM package name</th>
+  </tr>
+  <tr>
+    <td>Amazon EC2</td>
+    <td>foreman-ec2</td>
+  </tr>
+  <tr>
+    <td>Google compute engine</td>
+    <td>foreman-gce</td>
+  </tr>
+  <tr>
+    <td>Libvirt</td>
+    <td>foreman-libvirt</td>
+  </tr>
+  <tr>
+    <td>oVirt / RHV</td>
+    <td>foreman-ovirt</td>
+  </tr>
+  <tr>
+    <td>VMware</td>
+    <td>foreman-vmware</td>
+  </tr>
+</table>
+
+After installing new provider, restart the Foreman web server by running
+
+```
+service foreman restart
+```
+
+Due to lack of packaging capacity, for Debian based systems, we bundle all core
+providers into the main Foreman package. For more information, check out [manual](https://theforeman.org/manuals/latest/#5.2ComputeResources)
```

Comment:
I don't think this is true? At least, we create separate binary packages from the same source package. The installer also treats RPMs and debs the same.

Reply:
I guess no longer the truth, I had some text from 2 years ago :-) I found the deb packages now, it seems we started building them from 1.16, in 1.15 it [was still foreman-compute](http://deb.theforeman.org/pool/jessie/1.15/f/foreman/)